tck/README.adoc

```diff
@@ -101,3 +112,28 @@ For this reason, the rows of the expected results table are to be considered a s
 In order to implement the Cypher TCK, you will have to retrieve the full suite of TCK feature files, hosted at this GitHub repository, https://github.com/opencypher/openCypher/tree/master/tck/features[in this subdirectory].
 
 // TODO: Mention Cucumber ?
+
+[[errors]]
+=== Cypher errors
+
+The `Then` step used to specify expected errors from running a given invalid query follows this schematic setup:
+
+ Then a TYPE should be raised at TIME: DETAIL
+
+TYPE will be one of the following error types:
+
+// these are initially taken from Neo4j kernel's Status.Statement class' ClientErrors
+- SyntaxError                    "The statement contains invalid or unsupported syntax."
+- SemanticError                  "The statement is syntactically valid, but expresses something that the database cannot do."
+- ParameterMissing               "The statement is referring to a parameter that was not provided in the request."
+- ConstraintVerificationFailed   "A constraint imposed by the statement is violated by the data in the database."
+- EntityNotFound                 "The statement is directly referring to an entity that does not exist."
```

Comment:
"The statement is directly referring to an entity that does not exist." -> "The statement refers to a non-existent entity."
Is there any reason to include "directly" in this? I think it is self-evident that it does refer 'directly'...